service-front/module/Application/src/Helpers/LpaFormHelper.php

```diff
@@ -254,23 +253,25 @@ private function checkLpaNotAdded(string $lpa, array $detailsData): bool
         return true;
     }
 
-    public function lpaIdentitiesMatch(array $lpas, string $type): bool
+    public function lpaIdentitiesMatch(array $lpas, PersonType $personType): bool
     {
         if (count($lpas) == 1) {
             return true;
         }
 
-        $name = $lpas[0]['opg.poas.lpastore'][$type]['firstNames'] . " " .
-            $lpas[0]['opg.poas.lpastore'][$type]['lastName'];
+        $personTypeValue = $personType === PersonType::CertificateProvider ? 'certificateProvider' : 'donor';
```

Comment:
Nit, and totally ignorable if you don't agree, but I think "key" is a more accurate description of this variable
```suggestion
        $personTypeKey = $personType === PersonType::CertificateProvider ? 'certificateProvider' : 'donor';
```